constellations/gemini-user/src/account/payment.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+require("./account.sass");
```

Comment:
Similarly I think you should split your sass files. Keep each of these components isolated. Their JS files should be separate, and so should their CSS as well.